.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,74 @@
+on:
+  workflow_run:
+    workflows:
+      - Build [Docs]
+    types:
+      - completed
+
+defaults:
+  run:
+    shell: bash
+
+# We use a separate workflow so we don't leak deployment keys in the PR workflows.
+# Keys are tied to the 'docs-deploy' environment, which can only be accessed on the 'master' branch.
+
+# IMPORTANT: You have to merge changes to this file to 'master' before they'll be executed.
+
+# NOTE: We do not checkout any repository (ours or theirs):
+#       - Avoids Remote Code Execution (if we check out their branch)
+#       - Avoids user confusion when changes they make are not reflected (such as changing the Taskfile.yml)
+#       - GitOps based tools (wrangler) need to be configured explicitly
+
+# secrets:
+# DOCS_CLOUDFLARE_PAGES_API_TOKEN::
+# - Grants edit access to Cloudflare Pages on the Datapane Account
+# - TTL of 2 months
+# - instructions here: https://developers.cloudflare.com/api/tokens/create/
+
+# Future:
+# - implement 'workflow_dispatch' so we can manually trigger deploys
+# - implement Github Deployments to associate a deployed page with a PR (better reviews)
+# - Pages Cleanup for merged PR
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: github.event.workflow_run.conclussion == 'success'
```

Comment:
sp - `conclusion`

Reply:
Yep, this might have been why `act` was being silent for me 🤦

Reply:
ah - hopefully so! :)